e2e/windows_vmss_template.json

```diff
@@ -6,9 +6,9 @@
             "defaultValue": "",
             "type": "string"
         },
-        "galleries_windowsabgallery_externalid": {
-            "defaultValue": "/subscriptions/a15c116e-99e3-4c59-aebc-8f864929b4a0/resourceGroups/akswinvhdbuilderrg/providers/Microsoft.Compute/galleries/AKSWindows",
-            "type": "string"
+        "galleries_AKSWindows_externalid": {
```

Comment:
do we still need this parameter?

Reply:
yes, I set the value of it in e2e-windows-scenario.sh